hbw/config/config_run2.py

```diff
@@ -577,7 +582,7 @@ def make_jme_filename(jme_aux, sample_type, name, era=None):
     cfg.x.keep_columns = DotDict.wrap({
         "cf.MergeSelectionMasks": {
             "mc_weight", "normalization_weight", "process_id", "category_ids", "cutflow.*",
-            "HbbJet.n_subjets", "HbbJet.n_separated_jets", "HbbJet.max_dr_ak4",
+            "HbbJet.n_subjets", "HbbJet.n_separated_jets", "HbbJet.max_dr_ak4","m_ll", "channle_id",
```

Comment:
Hi Lara, thanks for this PR! I let @mafrahm or @apaasch review it more in detail and see whether it can be merged. I just wanted to point out that here there's likely a typo `channle_id` ---> `channel_id`